Altis_Life.Altis/stringtable.xml

```diff
@@ -7971,7 +7971,7 @@
             <Czech>Tato osoba buď respawned nebo již byl přijat.</Czech>
             <Spanish>Esta persona ya reapareció o fue revivida.</Spanish>
             <Russian></Russian>
-            <French>Cette personne a peut-être déjà réapparu ou a déjà été réanimée.</French>
+            <French>Cette personne a peut-être fait réapparition ou a déjà été réanimée.</French>
```

Comment:
Not good, that's linked to an action you can make in the game, why not keeping it RP ?